tools/log-collector/helper.go

```diff
@@ -145,7 +145,7 @@ func filterTvkCSV(csvObjects unstructured.UnstructuredList) unstructured.Unstruc
 }
 
 // filterRelatedCRD returns list of crds created by given set of groups
```

Comment:
comment > func name not update. Always use `refactor>rename` instead of manually changing names

Reply:
resolved